packages/l/logiops/package.yml

```diff
@@ -0,0 +1,24 @@
+name       : logiops
+version    : 0.3.4
+release    : 1
+source     :
+    - https://github.com/PixlOne/logiops/releases/download/v0.3.4/logiops-v0.3.4.tar.gz : 83923b303df6ae65e5c537854eaf4e78f0d2ae2871f74b54392462fe035e70de
+homepage   : https://github.com/PixlOne/logiops
+license    : GPL-3.0-or-later
+component  : system.utils
+summary    : Unofficial logitech driver and daemon
+description: |
+    This is an unofficial driver for Logitech mice and keyboard.
+builddeps  :
+    - cmake
+    - glib2-devel
+    - libconfig-devel
+    - libevdev-devel
+    - libgudev-devel
+setup      : |
+    %cmake_ninja \
+        -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
Our `cmake` macros set the build type to `RelWithDebugInfo`. This is so that the debug symbols can be stripped to their own package that users can install if they experience a crash and need to provide a backtrace, for example.